PrintJson.py

```diff
@@ -5,7 +5,7 @@
 from dotenv import load_dotenv
 
 #load in environment variables
-load_dotenv()
+load_dotenv(dotenv_path="CS125.env")
```

Comment:
What's the reason behind this change?

Reply:
same here as the previous files where if you renamed `CS125.env`, it should fix the `load_dotenv()`